$NetBSD$

Put a more likely to be correct path in the example config.

--- hercules.cnf.orig	2020-02-09 16:27:07.621576951 +0000
+++ hercules.cnf
@@ -4,6 +4,7 @@
 
   MSGLEVEL  VERBOSE                   # normal/verbose/debug
 # MODPATH   /usr/local/lib/hercules   # Where to search for modules
+  MODPATH   /usr/pkg/lib/hercules     # Where to search for modules
 
 
 #-------------------------------------------------------------------------------
